qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/CreateIdentityUITest.java

```diff
@@ -139,6 +143,31 @@ public void addIdentity1Test() throws Throwable {
         closeDrawer(R.id.drawer_layout);
     }
 
+    @Test
+    public void logoutTest() {
+        openDrawer(R.id.drawer_layout);
+        onView(withId(R.id.accountName))
+                .check(matches(isDisplayed()))
+                .perform(click());
+        onView(withText("logout"))
+                .check(matches(isDisplayed()))
+                .perform(click());
+        onView(withId(R.id.bt_login)).check(matches(isDisplayed()));
+        assertHasNoCredentials();
+        assertHasNoIdentities();
+    }
+
+    private void assertHasNoIdentities() {
+        removeAllIdentities();
+        assertEquals(0, queryIdentities().size());
```

Comment:
If you remove all identities directly in the service, of course there are no identities left. Why is this in the logoutTest?